src/graphics/workload.rs

```diff
@@ -0,0 +1,30 @@
+use wgpu::CommandEncoder;
+
+use super::*;
```

Comment:
Maybe `use super::Context;` instead of including the whole project?

In fact, maybe it's more of a question of programming style. I never write `use super::*;` except in unit tests. I prefer to know exactly what is included and from where. I may not have much of a reason though, only an opinion :D

Reply:
I do it because of the simplicity, but I do agree that some information is lost and it can get messy if we go too deep like here. In some cases, it makes sense because a submodule shares a namespace with its parent. I'll address this.